ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -571,4 +573,11 @@ QtObject {
     function hex2Eth(value) {
         return globalUtils.hex2Eth(value)
     }
+
+    readonly property Connections connections: Connections {
+      target: communitiesModuleInst
+      function onImportingCommunityStateChanged(communityId, state, errorMsg) {
+          root.importingCommunityStateChanged(communityId, state, errorMsg)
+      }
+    }
```

Comment:
I've refactored this to use a pattern I've learned from @osmaczko 
Instead of having a `Connection` with a target of a module instance straight on the view, we make the connection inside the store and then have the view  connect to the store instead (so it doesn't know anything about the module instance behind the scenes).

The reason I've refactored it here is because I needed the same in another store, so I wanted those places to be consistent.